earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,84 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
-
-task openGeeCommonDeb (type: Deb) {
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
     packageName = 'opengee-common'
+    release = "1.${rpmPlatformString}"
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Third-party libraries bundled with Open GEE'
+    // The package build failed when packageDescription was copied here from openGeeCommonRpm, so it is omitted for now
```

Comment:
@mrstephen: Perhaps, the description may not have been formatted properly.  What error did you get?

Reply:
![build failure](https://user-images.githubusercontent.com/12215726/38418256-a85c406a-3969-11e8-88b7-70a9e8e76bb4.PNG)

This is the error I see. I had to track it to the packageDescription property by trial and error. 
As for the format, I copied and pasted it directly from the `openGeeCommonRpm` task, so I was a little surprised it didn't build, because `openGeeCommonRpm` builds with no problem. 

I tried formatting it more like the the `license` property (each line enclosed in single quotes and joined by a +), but that didn't help. I also tried just setting it to `openGeeCommonRpm.packageDescription`, but that similarly failed. 

I built it with `scons -j8 release=1 package_install`. Is there a flag I can pass into that call to produce more detailed error messages?